smt/expr.cpp

```diff
@@ -731,6 +731,80 @@ expr expr::fdiv(const expr &rhs) const {
   return Z3_mk_fpa_div(ctx(), rm, ast(), rhs());
 }
 
+expr expr::isNaN() const {
```

Comment:
please keep the functions in the same order as in the header

Reply:
Sure, fixed :)